model-optimizer/mo/moc_frontend/pipeline.py

```diff
@@ -24,7 +24,7 @@ def moc_pipeline(argv: argparse.Namespace):
         str(fem.get_available_front_ends())))
     log.debug('Initializing new FE for framework {}'.format(argv.framework))
     fe = fem.load_by_framework(argv.framework)
-    input_model = fe.load_from_file(argv.input_model)
+    input_model = fe.load(argv.input_model)
 
```

Comment:
don't we need a binpath (or weight_path) here?

Reply:
We can load from just a model path, bin file assumed to lie near the model